### DISCUSSION TITLE
Get count of relationship (one to many)

### DISCUSSION THREAD

**Discussion by ragokan** [prisma/prisma#12834] [Q&A] [answered]
Hello, how can I get the count of relationship field?
For example, I want to get posts count of a user but I can't do this;

```ts
const users = await prisma.findMany({ include: { posts: { select: { _count : true } } } })
```

**Comment by austincrim** [comment, marked as answer]
Hey @ragokan 👋 ,

You will want something like [this](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#select-a-_count-of-relations):
```ts
await prisma.user.findMany({
  where: {
    id: 1
  },
  select: {
    _count: {
      select: {
        posts: true
      }
    }
  }
})
```

Let me know if that works for you!

**Reply by ragokan** [reply]
Ah, yes. Thank you, this works!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
